pkg/storage/mocks/slow_storage.go

```diff
@@ -0,0 +1,123 @@
+package mocks
+
+import (
+	"context"
+	"time"
+
+	"github.com/openfga/openfga/pkg/storage"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+)
+
+// SlowDataStorage is a proxy to the actual ds except the Reads are slow time by the readTuplesDelay
+// This allows simulating list objection condition that times out
+type SlowDataStorage struct {
```

Comment:
```suggestion
// SlowDataStorage is a proxy to the actual ds except the Reads are slow time by the readTuplesDelay
// This allows simulating list objection condition that times out
type slowDataStorage struct {
```

no need to export this struct